### PR TITLE
Release v0.23.0-rc.1

### PR DESCRIPTION
**Pull request by SuperQ**
BREAKING CHANGES:

This version of the exporter introduces a new configuration file format. This new format separates the walk and metric mappings from the connection and authentication settings. This allows for easier configuration of different auth params without having to duplicate the full walk and metric mapping.

See auth-split-migration.md for more details.

* [CHANGE] Split config of auth and modules #859
* [FEATURE] Add support for parsing SNMP transport from target #914
* [ENHANCEMENT] Improved Lookup process for label information #908
* [BUGFIX] Fix metrics path not using command-line argument value #904

### CODE REVIEW

**Review by roidelapluie — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-## 0.23.0-rc.0 / 2023-07-04
+## 0.23.0-rc.1 / 2023-07-04
```
Comment (roidelapluie) [inline CHANGELOG.md:1]: Jul 20 ?

Comment (SuperQ) [reply CHANGELOG.md]: Fixed

**Comment by RichiH** [comment]
Would it make sense to generate the version key by default but only INFO or WARN when it's missing?

I still believe that it's a good idea long term, even if we can't rely on it existing as of today.

**Pull request merged**